extensions/wikia/SemanticMediaWiki/includes/storage/SQLStore/SMW_SQLStore3_Writers.php

```diff
@@ -777,7 +777,7 @@ protected function updateRedirects( $subject_t, $subject_ns, $curtarget_t = '',
 			if ( $smwgEnableUpdateJobs && ( $smwgQEqualitySupport != SMW_EQ_NONE ) ) {
 				// entries that refer to old target may in fact refer to subject,
 				// but we don't know which: schedule affected pages for update
-				$jobs = array();
+				$titles = array();
```

Comment:
It does. This method is going to be thoroughly refactor. That was just to verify the concept and possibly mitigate performance problem.